translations/translation_br.xml

```diff
@@ -15,65 +15,65 @@
-    <text name="CP_fieldWorkJobParameters_startAt_bunkerSilo" text="modo silo bunker"/>
-    <text name="CP_fieldWorkJobParameters_startAt_siloLoader" text="Modo carregar silo"/>
-    <text name="CP_fieldWorkJobParameters_startAt_unload" text="descarregando colheitadeira"/>
-    <text name="CP_fieldWorkJobParameters_startAt_unload_siloLoader" text="Descarregar carregador de silo"/>
-    <text name="CP_fieldWorkJobParameters_startAt_bales" text="fardos coletar/embalar"/>
-    <text name="CP_fieldWorkJobParameters_subTitle_laneOffset" text="Distância da fileira:"/>
-    <text name="CP_fieldWorkJobParameters_laneOffset_title" text="Distância da fileira:"/>
-    <text name="CP_fieldWorkJobParameters_laneOffset_tooltip" text="Distância da fileira"/>
-    <text name="CP_fieldWorkJobParameters_laneOffset_center" text="centro"/>
-    <text name="CP_fieldWorkJobParameters_laneOffset_left" text="esquerda(1)"/>
-    <text name="CP_fieldWorkJobParameters_laneOffset_right" text="direita(1)"/>
-    <text name="CP_fieldWorkJobParameters_laneOffset_left_2" text="esquerda(2)"/>
-    <text name="CP_fieldWorkJobParameters_laneOffset_right_2" text="direita(2)"/>
+    <text name="CP_jobParameters_fieldPosition_title" text="Posição no campo:"/>
+    <text name="CP_jobParameters_bunkerSiloPosition_title" text="Posição do silo:"/>
+    <text name="CP_jobParameters_parkPosition_title" text="Posição de estacionamento:"/>
+    <text name="CP_jobParameters_subTitle_startAt" text="Iniciar em:"/>
+    <text name="CP_jobParameters_startAt_title" text="Iniciar em:"/>
+    <text name="CP_jobParameters_startAt_tooltip" text="Onde começar o trabalho no campo"/>
+    <text name="CP_jobParameters_startAt_nearest" text="próximo waypoint"/>
+    <text name="CP_jobParameters_startAt_first" text="primeiro waypoint"/>
+    <text name="CP_jobParameters_startAt_last" text="último waypoint"/>
+    <text name="CP_jobParameters_startAt_bunkerSilo" text="Modo silo de bunker"/>
+    <text name="CP_jobParameters_startAt_siloLoader" text="Modo carregar silo"/>
+    <text name="CP_jobParameters_startAt_unload" text="descarregar colheitadeira"/>
+    <text name="CP_jobParameters_startAt_unload_siloLoader" text="Descarregar carregador de silo"/>
+    <text name="CP_jobParameters_startAt_bales" text="coletar/embalar fardos"/>
+    <text name="CP_jobParameters_subTitle_laneOffset" text="Distância da fileira:"/>
+    <text name="CP_jobParameters_laneOffset_title" text="Distância da fileira:"/>
+    <text name="CP_jobParameters_laneOffset_tooltip" text="Distância da fileira"/>
+    <text name="CP_jobParameters_laneOffset_center" text="centro"/>
+    <text name="CP_jobParameters_laneOffset_left" text="esquerda(1)"/>
+    <text name="CP_jobParameters_laneOffset_right" text="direita(1)"/>
+    <text name="CP_jobParameters_laneOffset_left_2" text="esquerda(2)"/>
+    <text name="CP_jobParameters_laneOffset_right_2" text="direita(2)"/>
```

Comment:
You seem to have a very old version.
Revert these changes, as the correct one is "CP_fieldWorkJobParamater..."
Just compare it to the other translations e.g. this https://github.com/Courseplay/Courseplay_FS22/blob/ab4e01d97a9c2f78e5b2a27fdb75ecb312f00144/translations/translation_en.xml#L33

Reply:
thanks for the review